src/main/java/seedu/address/model/expenditure/Amount.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.model.expenditure;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Amount {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Expense amount should only contain positive numbers, and it should be formatted to accept 2 decimal places";
+    public static final String VALIDATION_REGEX = "^\\s*(?=.*[1-9])\\d*(?:\\.\\d{1,2})?\\s*$";;
+    public final String amount;
```

Comment:
Do you think it's better to use double instead of string? Similar for the Date object, should we use LocalDate?